src/ducks/balance/components/GroupPanel.jsx

```diff
@@ -45,13 +45,7 @@ const GroupPanelSummary = withStyles(() => ({
 
 const GroupPanelExpandIcon = React.memo(function GroupPanelExpandIcon() {
   return (
-    <span className="u-click-xl">
```

Comment:
u-click-xl should be replaced by u-click-200 ?

Reply:
In fact, I checked and it was useless to have this class here, the clickable area is already big.